Compression.md

```diff
@@ -0,0 +1,145 @@
+## WebSocket Compression
+WebSocket Compression, defined by [RFC7692](https://tools.ietf.org/html/rfc7692) allows WebSocket clients to send and receive compressed data on a WebSocket connection. Compression reduces the total wire-level payload of a WebSocket connection, possibly resulting in an improved throughput.
+
+This document discusses the implementation of WebSocket Compression in [Kitura-WebSocket-NIO](https://github.com/IBM-Swift/Kitura-WebSocket-NIO), which is an implementation of the [Kitura-WebSocket](https://github.com/IBM-Swift/Kitura-WebSocket) API using [swift-nio](https://github.com/apple/swift-nio).
```

Comment:
A minor question - do we usually refer to swift-nio or should we be using the name SwiftNIO here (the link to the github repo is fine either way)?

Reply:
SwiftNIO

Reply:
Replaced all instances of `swift-nio` by `SwiftNIO`